src/styles/themes/_definitions.scss

```diff
@@ -88,6 +88,18 @@ $eightfold-logo-gradient-dark: linear-gradient(
         #1abbcd 50.02%,
         #6fc6aa 100.03%
     );
+$expandable-header-gradient: linear-gradient(
+    270deg,
+    hsla(0, 0%, 100%, 0) 22.25%,
+    hsla(0, 0%, 100%, 0.995) 41.17%,
+    #fff 80.83%
```

Comment:
this needs to be theme var driven

Reply:
these are directly from design and we don't have such colors.

Reply:
we do have a #fff color var (its background or something). We should use it. If someone implements a dark color theme for instance, we don't want this remaining white.